dynamo/preprocessing/preprocess.py

```diff
@@ -1626,9 +1626,14 @@ def _infer_experiment_type(adata):
     adata = filter_cells(adata, keep_filtered=keep_filtered_cells, **filter_cells_kwargs)
     logger.info(f"{adata.obs.pass_basic_filter.sum()} cells passed basic filters.")
 
+    filter_genes_layer = None
```

Comment:
We should  always using all layers for filter_genes_kwargs

Reply:
the feature selection layer is only used to select the feature genes; should not be used to filter genes.